sdk_lib/make_chroot.sh

```diff
@@ -21,7 +21,7 @@ fi
 
 # Check if the host machine architecture is supported.
 ARCHITECTURE="$(uname -m)"
-if [[ "$ARCHITECTURE" != "x86_64" ]]; then
+if [[ "$ARCHITECTURE" != "x86_64" && "$ARCHITECTURE" != "ppc64le" ]]; then
   echo "$SCRIPT_NAME: $ARCHITECTURE is not supported as a host machine architecture."
   exit 1
```

Comment:
Do you really intend to get the SDK hosted on ppc64le?  If so, I would say leave this until all the support needed to host on ppc64le is ready.  The enabling of it should be the last thing.